src/main/features/claim/helpers/eligibility/validationGroups.ts

```diff
@@ -1,4 +1,5 @@
 export class ValidationGroups {
+  static readonly CITIZEN_CLAIMANT: string = 'citizen_claimant'
```

Comment:
Without any context, what does a name like `CITIZEN_CLAIMANT` tell you? My first thought would be that it means that the claimant is a citizen. Also, please use consistent naming - in other places it's called `claimOnBehalf`, so I'd just use that.

Reply:
Citizen claimant is valid classification for an eligible claim.
Claim on behalf is not a valid classification, it is a discriminator for eligibility. 
I agree with your point about consistency but CLAIM_ON_BEHALF does not fit in with how the ValidationGroup should be named.

ValidationGroup.ts is the opposite of notEligibleReasons.ts. 
So using a different example:
Valid = OVER_18 - "I am OVER_18. Yes - you can use the service" - *Valid reason is true*
Invalid = UNDER_18 - "I am UNDER_18. No - I  can not use the service" - *Invalid reason is true*

If we applied the same logic, we would have
Valid = CLAIM_ON_BEHALF- "This is a CLAIM_ON_BEHALF. No - but you cant use the service" - *valid reason is FALSE*
Valid = CITIZEN_CLAIMANT- "This is a CITIZEN_CLAIMANT. Yes - as nobody is claiming on your behalf, you are eligible" *valid reason is TRUE*

Invalid = CLAIM_ON_BEHALF - "This is a CLAIM_ON_BEHALF. Yes. You can not use the service - *invalid reason is true*

Reply:
Talked to Damian about it and its been changed now. I took validationGroups to mean reasons why it was valid. Just a misunderstanding on my part. Thanks for catching this